R/get_predicted.R

```diff
@@ -480,16 +486,18 @@ get_predicted.bife <- function(x,
       stringsAsFactors = FALSE
     )
     iter <- as.data.frame(iter_stacked)
-    names(iter) <- paste0("iter_", names(iter))
     # outcome with a single level
   } else {
     # .get_predicted_boot already gives us the correct observation ~ draws format
     if (is.null(colnames(iter)) || !all(grepl("^iter", colnames(iter)))) {
       iter <- as.data.frame(t(iter))
-      names(iter) <- gsub("^V(\\d+)$", "iter_\\1", names(iter))
+
```

Comment:
This regex wasn't working as expected, giving results like `iter_V1`, `iter_V2` instead of `iter_1`

Reply:
The issue here would be that column names originally look like "iter_V1" rather than "V1", so nothing is being modified because there are no matches.  

The regex should change to 

`gsub("^(?:iter_)*V(\\d+)+", "iter_\\1", names(iter))`

Reply:
I wonder if that's more efficient than simply assigning a new vector of names, it's not like we expect anything else than iter_1:ncol()